control/control.yaml

```diff
@@ -40,6 +40,9 @@ components:
             The names of flows to which the transmit state will be applied to.
             If the list of flow_names is empty or null the state will be applied
             to all configured flows.
+
```

Comment:
@ajbalogh should it have the detailing about metrics?

Reply:
@shramroy the first comment in the PR has uses cases and commit 2d61746 has more context information regarding clearing of metrics